src/main.rs

```diff
@@ -80,8 +80,16 @@ fn try_main() -> Result<(), Box<dyn std::error::Error>> {
         println!("{}", solver::solution_to_string(solution)?);
     }
 
-    if let SolveStatus::TimedOut { elapsed } = solve_result.status {
-        eprintln!("⚠️  Timed out after {:.1}s; only partial solutions returned", elapsed.as_secs_f64());
+    match solve_result.status {
+        SolveStatus::TimedOut { elapsed } => {
+            eprintln!("⚠️  Timed out after {:.1}s; only partial solutions returned", elapsed.as_secs_f64());
```

Comment:
perhaps "some solutions may not have been returned" instead of "only partial solutions returned" (for two reasons: (1) technically, by chance, it could have returned all of the solutions and (2) "partial solutions" reads to me as if each individual solution is somehow partial, rather than the _list_ of solutions being incomplete (i.e. partial))